$NetBSD: patch-CMakeLists.txt,v 1.3 2013/03/26 11:08:18 othyro Exp $

Fix INSTALL_MANDIR overwriting itself and don't hardcode LIB_SUFFIX for NetBSD.

--- CMakeLists.txt.orig	2012-12-11 18:03:50.000000000 +0000
+++ CMakeLists.txt
@@ -43,14 +43,13 @@ if(WIN32)
     set(STATEDIR "var" CACHE PATH "state directory")
 else()
     if (NOT LIB_SUFFIX)
-        if (NOT EXISTS /etc/debian_version)
+        if (NOT EXISTS /etc/debian_version AND NOT ${PLATFORM_ID} STREQUAL "NetBSD")
             if (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64")
                 set(LIB_SUFFIX 64)
             endif()
         endif()
     endif()
     set(INSTALL_SBINDIR sbin)
-    set(INSTALL_MANDIR share/man)
     set(INSTALL_LOCALE share/locale)
     set(INSTALL_DOCDIR share/doc/${PROJECT_NAME})
     set(INSTALL_INFODIR share/info)
